docs/data-sources/secret_version.md

```diff
@@ -5,82 +5,94 @@ page_title: "Scaleway: scaleway_secret_version"
 
-- `secret_id` - (Optional) The Secret ID associated wit the secret version.
-  Only one of `secret_id` and `secret_name` should be specified.
+- `secret_id` - (Optional) The ID of the secret associated with the secret version. Only one of `secret_id` and `secret_name` should be specified.
 
-- `secret_name` - (Optional) The Name of Secret associated wit the secret version.
+- `secret_name` - (Optional) The name of the secret associated with the secret version.
   Only one of `secret_id` and `secret_name` should be specified.
 
-- `revision` - The revision for this Secret Version. Alternative values (ex: `latest`) can be found in [API documentation](https://www.scaleway.com/en/developers/api/secret-manager/#path-secret-versions-access-a-secrets-version-using-the-secrets-id)
-
-- `region` - (Defaults to [provider](../index.md#region) `region`) The [region](../guides/regions_and_zones.md#regions)
-  in which the resource exists.
+- `revision` - The revision for this secret version. Refer to alternative values (ex: `latest`) in the [API documentation](https://www.scaleway.com/en/developers/api/secret-manager/#path-secret-versions-access-a-secrets-version-using-the-secrets-id)
 
-- `project_id` - (Optional) The ID of the project the Secret version is associated with.
+- `project_id` - (Optional) The ID of the Scaleway Project associated with the secret version.
 
-## Data
+## Data information
 
-Note: This Data Source give you **access** to the secret payload encoded en base64.
+Note: This data source provides you with access to the secret payload, which is encoded in base64.
 
-Be aware that this is a sensitive attribute. For more information,
+Keep in mind that this is a sensitive attribute. For more information,
 see [Sensitive Data in State](https://developer.hashicorp.com/terraform/language/state/sensitive-data).
 
-~> **Important:**  This property is sensitive and will not be displayed in the plan.
+~> **Important:**  This property is sensitive and will not be displayed in the Terraform plan, for security reasons.
 
 ## Attributes Reference
 
-In addition to all arguments above, the following attributes are exported:
+This section lists the attributes that are exported by the `scaleway_secret_version` data source. These attributes can be referenced in other parts of your Terraform configuration:
 
-- `description` - (Optional) Description of the secret version (e.g. `my-new-description`).
-- `data` - The data payload of the secret version. more on the [data section](#data)
-- `status` - The status of the Secret Version.
-- `created_at` - Date and time of secret version's creation (RFC 3339 format).
-- `updated_at` - Date and time of secret version's last update (RFC 3339 format).
+- `description` - (Optional) The description of the secret version (e.g. `my-new-description`).
+- `data` - The data payload of the secret version. This is a sensitive attribute containing the secret value. Learn more in the [data section](#data).
```

Comment:
`- `data` - The data payload of the secret version. This is a sensitive attribute containing the secret value. Learn more in the [data section](#data-information)`

Reply:
This should fix the pipeline :)